index.html

```diff
@@ -1,52 +1,314 @@
-        <section class="intro">
-            <h1 class="intro_greeting">I'm George<br> Nice to see you!</h1>
-            <p class="intro_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <div class="connect">
-                <div class="connect_text">
-                    <p>LET'S CONNECT<p>    
-                </div>
-            </div>
-            <ul class="socials">
-                <li>
-                    <a href="https://twitter.com/GeorgeMsapenda" target="_blank">
-                        <img src="images/Twitter icon.png" alt="Twitter link">
-                    </a>
-                </li>
-                <li>
-                    <a href="linkedin.com/in/george-m-sapenda-593750201" target="_blank">
-                        <img src="images/Linkedin icon.png" alt="Linkedn link">
-                    </a> 
-                </li>
-                 <li>
-                    <a href="https://github.com/c00p75" target="_blank">
-                        <img src="images/Github icon.png" alt="Github link">
-                    </a>
-                </li>                                
-            </ul>
-        </section>
-    </main>
+	<main class="container">
+		<header class="header">
+			<nav class="navigation">  
+				<div class="logo"> 
+					<a href="#">George M'sapenda</a>
+				</div>
+					<button class="btn">
+						<img src="/images/Icon - Menu.png" alt="menu bar icon">
+					</button>                    
+				</div>
+			</nav>
+		</header>
+
+			<section class="intro">
+					<h1 class="intro_greeting">I'm George<br> Nice to see you!</h1>
+					<p class="intro_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+					<div class="connect">
+						<div class="connect_text">
+							<p>LET'S CONNECT<p>    
+						</div>
+					</div>
+					<ul class="socials">
+						<li>
+							<a href="https://twitter.com/GeorgeMsapenda" target="_blank">
+								<img src="images/Twitter icon.png" alt="Twitter link">
+							</a>
+						</li>
+						<li>
+							<a href="linkedin.com/in/george-m-sapenda-593750201" target="_blank">
+								<img src="images/Linkedin icon.png" alt="Linkedn link">
+							</a> 
+						</li>
+							<li>
+								<a href="https://github.com/c00p75" target="_blank">
+									<img src="images/Github icon.png" alt="Github link">
+								</a>
+						</li>                                
+					</ul>
+			</section>
+
+			<section class="Works">
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio.png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Tonic</p>
```

Comment:
:red_circle: This should be a `hx` tag. A `p` tag is not semantically appropriate. Please change it to a heading